src/System.Net.Http/src/System/Net/Http/Unix/CurlCallbacks.cs

```diff
@@ -204,15 +206,25 @@ private static bool TryParseStatusLine(HttpResponseMessage response, string resp
                 return false;
             }
 
+	    //Clear the header if status line is recieved again. This signifies that there are multiple response headers (like in redirection).
```

Comment:
Nit: indentation looks wrong. The other comments also have a space between the // and the start of the text.